docs/source/profiler.rst

```diff
@@ -757,6 +760,7 @@ Below is an breakdown of all the options.
         the default labeler 
       * max_sample_size - (Int) The max number of samples for the data 
         labeler
+	* **row_statistics** - (Boolean) Option to enable/disable row statistics calculations
```

Comment:
does this tabbing need to be fixed?

Reply:
+1 one indentation... I would think this shouldn't go in the `data_labeler` section of options, no?